x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/CategorizationIT.java

```diff
@@ -285,13 +286,11 @@ public void testNumMatchesAndCategoryPreference() throws Exception {
 
         CategoryDefinition category1 = categories.get(0);
         assertThat(category1.getNumMatches(), equalTo(2L));
-        long[] expectedPreferenceTo = new long[]{2L, 3L, 4L, 5L, 6L};
-        assertThat(category1.getPreferredToCategories().length, equalTo(5));
```

Comment:
I removed the assertion on the length because in the case where the results are not as expected it obscures the useful information of what the actual results were.  Equal length should be implied by the equalTo assertion on the line below.